doc/dev/design/ui_goals.rst

```diff
@@ -0,0 +1,57 @@
+Ceph Dashboard Design Goals
+===========================
+
+.. note:: this document is intended to provide a focal point for discussing the overall design principles for mgr/dashboard
+
+Introduction
+############
+
+Most distributed storage architectures are inherently complex, and can present a management challenge to Operations teams who are
+typically stretched across multiple product and platform disciplines. In general terms, the complexity of any solution can have a 
+direct bearing on the operational costs incurred to manage it. The answer is simple...make it simple :)
+
+This document is intended to highlight Ceph Dashboard design goals which may help to  
+
+* reduce complexity
+* increase productivity
+* improve time-to-value
+* increase observability
+
+ 
+Understanding the Persona of the Target User
+############################################
+
+Ceph has historically been administered from the CLI. The CLI has always and will always offer the richest, most flexible way to 
+install and manage a Ceph cluster. Administrators who require and demand this level of control are unlikely to adopt a UI for anything
+more than a technical curiousity.
+
+The relevance of the UI is therefore more critical for a new SysAdmin, where it can help technology adoption and reduce the operational friction
+that is normally experienced when implementing a new solution.
+
+Understanding the target user persona is therefore a fundamental first step in design. Attempting to design a UI that meets the requirements of a 'seasoned'
+Ceph Administrator or Developer, and a relatively new SysAdmin is unlikely to satisfy either user group.
+
+Design Principles
+#################
+
+Key Principles
+--------------
+
+
+#. **Clarity and consistency**. The UI should ensure the data shown is unambiguous and consistent across different views
+#. **Data timeliness**. Data displayed in the UI must be timely. State information must always be reasonably recent for it to be relevant and acted upon with confidence.
```

Comment:
I would also add that the "age" of state information should be visible, so that the user is aware of data that has not been refreshed recently (e.g. because the operation to gather that data is too expensive or takes too much time to be run in a loop).

Reply:
+1000! We have also had a tendency to put UTC timestamps in the tables which is also not that helpful. All the mockups I'm doing show the time as age not stamp - i.e. "20s ago" instead of 2020-09-27T14:34:19.567291

How expensive the data is to collect is something we're trying to offoad to each host - and then it can be seconds not minutes old - since the scrape for the UI becomes asynchronous to the actual data gathering.